api/v1/swagger.yaml

```diff
@@ -41,8 +42,8 @@ basePath: /v1
 
 # =============================================================================
 schemes:
-  - https
   - http
+  - https
```

Comment:
It was a known issue for swagger 2.0 when i originally did this on trust. It means that it will default to http unless the page is served over https in which case it will serve over https. Shouldnt make any difference to how api working just for docs as schemes were set to http and https anyway to allow it to accept both.